addon/components/freestyle/usage/index.hbs

```diff
@@ -56,6 +56,32 @@
       {{/if}}
     {{/if}}
 
+    {{#if (has-block 'cssVars')}}
+      {{#if this.showApi}}
+        <div class="FreestyleUsage-cssVars">
+          <h3 class="FreestyleUsage-cssVarsTitle">CSS Variables API</h3>
+
+          <table class="FreestyleUsage-cssVarsTable">
+            <thead>
+              <tr>
+                <th>Variable</th>
+                <th>Description</th>
+                <th>Default</th>
+                <th>Value</th>
+              </tr>
+            </thead>
+            <tbody>
+              {{yield
+                (hash
+                  Basic=(component 'freestyle/usage/basic-css-variable' )
```

Comment:
I was thinking it might be cool to have variations that have interactive controls for different [CSS value data types](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Values_and_Units)